lib/src/live_chat.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:live_chat/src/live_chat_platform_interface.dart';
+
+class LiveChat {
+  static final LiveChat _singleton = LiveChat._internal();
+
+  factory LiveChat() {
+    return _singleton;
+  }
+
+  LiveChat._internal();
+
+  Future<void> openChatView({
+    required String licenseId,
+    required String username,
+    required String email,
+    String? groupId,
+    Map<String, String>? customParameters,
```

Comment:
Can't be Map<String, dynamic> ?

Reply:
@Reza-deriv the sdk restricts string for value as well.